### PR TITLE
Add LVI to changelog

### PR DESCRIPTION
**Pull request by radhikaj**
Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>

### CODE REVIEW

**Comment by BRMcLaren** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - OpenSSL engine support to oesign to allow signing keys via engines.
 - NuGet package validation using CI/CD.
-
+- Released packages include Load Value Injection(LVI) mitigated libraries and LVI mitigation build configuration.
```
Comment (mingweishih) [inline CHANGELOG.md]: Would "... and LVI mitigation build instructions." be better?

**Pull request merged**